backend/src/utils/db.py

```
@@ -0,0 +1,291 @@
+def get_active_positions():
+    conn = get_db_connection()
+    cursor = conn.cursor()
+    cursor.execute('SELECT * FROM positions')
+    rows = cursor.fetchall()
+    positions = []
+    for row in rows:
+        pos = dict(row)
+        if pos.get('metadata'):
+            try:
+                pos['metadata'] = json.loads(pos['metadata'])
+            except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)